setup.py

```diff
@@ -64,6 +64,7 @@ def read_requirements(filename):
     "jsonschema",
     "platformdirs",
     "tblib==1.7.0",
+    "pyvis==0.2.1",
```

Comment:
an alternative to this would be to import pyvis dynamically when needed, otherwise every consumer of the sdk will need to install pyvis even if they don't use this behavior.

example:
https://github.com/search?q=repo%3Aaws%2Fsagemaker-experiments%20get_module&type=code

Reply:
overall this looks good but this change could result in unforeseen side effects. for example if every consumer of the sagemaker sdk has to install pyvis what if that installation fails or is incompatible with some part of the consumer's environment.